src/index.js

```diff
@@ -18,7 +18,7 @@ const glamorous = createGlamorous(splitProps)
 Object.assign(
   glamorous,
   domElements.reduce((getters, tag) => {
-    getters[tag] = glamorous(tag)
+    getters[tag] = glamorous(tag, {displayName: `glamorous.${tag}`})
```

Comment:
Note this change is inspired by @liadbiz who found `glamorous(div)` confusing. I think this is less confusing because it looks like how you would code it: `glamorous.div` rather than `glamorous(div)`.